tests/test_entry_point.py

```diff
@@ -20,7 +20,7 @@
 
 STRIPPED_DOWN_AND_FULLY_DISCRETE_MODELS = [
```

Comment:
```suggestion
STRIPPED_DOWN_AND_DISCRETE_MODELS = [
```

But see below before changing anything.